gateway/grpc_test.go

```diff
@@ -31,26 +31,31 @@ import (
 
 // For gRPC, we should be sure that HTTP/2 works with Tyk in H2C configuration also for insecure grpc over http.
 func TestHTTP2_h2C(t *testing.T) {
+
 	defer ResetTestConfig()
 	var port = 6666
 
-	EnablePort(port, "h2c")
+	EnablePort(port, "http")
```

Comment:
changed as the protocol `user->tyk` is not necessarily the same as `tyk->upstream`. Users that already were using this feature need to be aware of this change